renderer/reducers/invoice.js

```diff
@@ -208,20 +198,18 @@ const ACTION_HANDLERS = {
 const invoiceSelectors = {}
 const invoiceSelector = state => state.invoice.invoice
 const invoicesSelector = state => state.invoice.invoices
-const invoicesSearchTextSelector = state => state.invoice.invoicesSearchText
 
-invoiceSelectors.invoiceModalOpen = createSelector(
-  invoiceSelector,
-  invoice => !!invoice
-)
+invoiceSelectors.invoice = invoiceSelector
 
 invoiceSelectors.invoices = createSelector(
   invoicesSelector,
-  invoicesSearchTextSelector,
-  (invoices, invoicesSearchText) =>
-    invoices.filter(invoice => invoice.memo && invoice.memo.includes(invoicesSearchText))
+  invoices => invoices
 )
 
+invoiceSelectors.invoiceModalOpen = createSelector(
+  invoiceSelector,
+  invoice => !!invoice
```

Comment:
Is it me who added this? I know that you prefer `Boolean`

Reply:
I think this has been here since the beginning of time. Updated